lib/repl.js

```diff
@@ -229,7 +229,7 @@ function REPLServer(prompt,
     let awaitPromise = false;
     const input = code;
 
-    if (/^\s*\{/.test(code) && /\}\s*$/.test(code)) {
+    if (/^\s*{/.test(code) && /\}\s*$/.test(code)) {
```

Comment:
What about the `}` in the second `test()`?

Reply:
Looks can change too. Wonder why Clion not suggest it. Anyway, I will apply the change.